modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -209,6 +209,11 @@ Events:     <11>
 <10> The pods on the node.
 <11> The events reported by the node.
 
+[NOTE]
+====
+The control plane label is not automatically added to newly created or updated master nodes. If you want to use the control plane label for your nodes, you can manually configure the label. See xref:../../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes] for more information about manually configuring labels.
```

Comment:
Think maybe the climbing path is wrong. 

```suggestion
The control plane label is not automatically added to newly created or updated master nodes. If you want to use the control plane label for your nodes, you can manually configure the label. See xref:../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes] for more information about manually configuring labels.
```

Reply:
Thanks! The path was the problem. I fixed it.